db/migrate/20240516101409_updateresourcepoolidentifierstomiqproductfeatures.rb

```diff
@@ -0,0 +1,63 @@
+class Updateresourcepoolidentifierstomiqproductfeatures < ActiveRecord::Migration[6.1]
+  class MiqProductFeature < ActiveRecord::Base; end
+
+  FEATURE_MAPPING_UPDATE = {
+    'resource_pool'           => 'resource_pool_cloud',
+    'resource_pool_view'      => 'resource_pool_cloud_view',
+    'resource_pool_show_list' => 'resource_pool_cloud_show_list',
+    'resource_pool_show'      => 'resource_pool_cloud_show',
+    'resource_pool_control'   => 'resource_pool_cloud_control',
+    'resource_pool_tag'       => 'resource_pool_cloud_tag',
+    'resource_pool_protect'   => 'resource_pool_cloud_protect',
+    'resource_pool_admin'     => 'resource_pool_cloud_admin',
+    'resource_pool_delete'    => 'resource_pool_cloud_delete'
+  }.freeze
```

Comment:
@Guddetisandeep since the existing product feature was for infrastructure providers we should update the existing records to _infra not _cloud